aiida/orm/nodes/node.py

```diff
@@ -547,7 +545,7 @@ def is_created_from_cache(self) -> bool:
         """
         kls = self.__class__.__name__
         warn_deprecation(
-            f'`{kls}.is_created_from_cache` is deprecated, use `{kls}.base.extras.is_created_from_cache` instead.',
+            f'`{kls}.is_created_from_cache` is deprecated, use `{kls}.base.caching.is_created_from_cache` instead.',
```

Comment:
Why does this, and `is_valid_cache`, not get re-routed via `__getattr__`, similar to everything else?

Reply:
Because it is a property that also has a setter and that doesn't work with the reroute through `__getattr__`.

Reply:
`is_created_from_cache` doesn't appear to have a setter though?

Reply:
True, will remove that and use the `__getattr__` for it